app/src/main/java/com/example/stretchy/features/datatransport/DataImporter.kt

```diff
@@ -1,5 +1,7 @@
 package com.example.stretchy.features.datatransport
 
 interface DataImporter {
-    suspend fun importData()
+    suspend fun importDataByAppending(data: String)
```

Comment:
can be simplified to `importByAppending`, no need to say `Data` once again, its in the interface name